frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -281,23 +277,25 @@ export function ActionFilterRow({
             onClick={() => {
                 duplicateFilter(filter)
             }}
-            className={`row-action-btn show-duplicabe`}
-            data-attr={'show-prop-duplicate-' + index}
+            style={{ display: 'flex', alignItems: 'center' }}
+            className={'row-action-btn'}
+            data-attr={`show-prop-duplicate-${index}`}
             title="Duplicate graph series"
         >
-            <CopyOutlined />
+            <IconCopy fontSize={'1.25em'} />
         </Button>
     )
 
     const deleteButton = (
         <Button
             type="link"
             onClick={onClose}
+            style={{ display: 'flex', alignItems: 'center' }}
             className="row-action-btn delete"
-            data-attr={'delete-prop-filter-' + index}
+            data-attr={`delete-prop-filter-${index}`}
             title="Delete graph series"
         >
-            <DeleteOutlined />
+            <IconDelete fontSize={'1.25rem'} />
```

Comment:
This one is bigger than the rest because it's `1.25rem` instead of `em`, is this intended?

Reply:
ooops nice catch :)